docs/man-openmpi/man3/MPI_Alloc_mem.3.rst

```diff
@@ -74,6 +74,34 @@ pointer object for this parameter. The provided argument should be a
 pointer to a pointer of arbitrary type (e.g., ``void **``).
 
 
+Fortran NOTES
+-------------
+
+The :ref:`MPI_Alloc_mem` calls requirse the use of the ``iso_c_binding`` module
+for due to the use of ``TYPE(C_PTR)``.
+
+.. code-block:: fortran
+
+   use iso_c_binding
+
+   type(c_ptr) :: alloc_ptr
+   integer :: size, ierr
+   double precision, pointer :: array(:,:)
+
+   ! A 2D array of 100 elements
+   size = 10 * 10
+   call MPI_Alloc_Mem(size * 8, MPI_INFO_NULL, alloc_ptr, ierr)
+
+   ! Point to the array
+   call c_f_pointer(alloc_ptr, array, [10, 10])
+
+   ! use array
+
```

Comment:
Did you mean to put some code here?  Or is this a "...code uses the array here..." kind of annotation?

Reply:
The latter, *now you can use the array*, but I can remove it :)

Reply:
I don't know offhand what we do with other examples, but perhaps something like this would be appropriate:

``` fortran
    ! ... use the array ...
```

And don't have 2 blank lines after it (just 1 blank line).

Reply:
Agreed, the 2 lines is a preference of mine to highlight the discontinuation of the code block, I will amend.